.docker/qgis.dockerfile

```diff
@@ -25,7 +28,7 @@ RUN cmake \
   -GNinja \
   -DCMAKE_BUILD_TYPE=Release \
   -DCMAKE_INSTALL_PREFIX=/usr \
-  -DWITH_DESKTOP=OFF \
+  -DWITH_DESKTOP=ON \
```

Comment:
I am a bit afraid to run out of time on Travis with this. 
I will enable the build on this branch to see what's going on.

The main issue it that enabling/disabling tests leads to a complete rebuild (because of add_definitions). I'm discussing with @m-kuhn to see if we can improve here (apparently yes)

Reply:
With `WITH_DESKTOP=OFF` the whole PR would makes no sense. Let me know if this is a blocker because if it is, I will go back to the original plan of putting this image in a separate repo.